### PR TITLE
CCTRI-2624 Version endpoint and remaining items

### PR DESCRIPTION
**Pull request by mstoro**

### CODE REVIEW

File: code/api/enrich.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def observe_observables():
         result = client.make_observe(observable)
         rules = result['data']['risk'].get('evidenceDetails')
         if rules:
+            if len(rules) > current_app.config['CTR_ENTITIES_LIMIT']:
+                rules = rules[:current_app.config['CTR_ENTITIES_LIMIT']].sort(key=lambda elem: elem['criticality'], reverse=True)
```
Comment (AnastasiiaRozlyvan) [inline code/api/enrich.py:37]: Does flake8 accept that length of line?

Comment (mstoro) [reply code/api/enrich.py]: Fixed it.

File: code/api/enrich.py [hunk 2]
```diff
@@ -33,6 +33,9 @@ def observe_observables():
         result = client.make_observe(observable)
         rules = result['data']['risk'].get('evidenceDetails')
         if rules:
+            if len(rules) > current_app.config['CTR_ENTITIES_LIMIT']:
+                rules = rules[:current_app.config['CTR_ENTITIES_LIMIT']].sort(
```
Comment (AnastasiiaRozlyvan) [inline code/api/enrich.py:37]: I'm not sure about this part of the code. So you first slice a certain number of rules and then sort them, right? But maybe you should sort them first, and then take, for example, 100 (or what limit the user will specify) of the most critical rules?

Comment (mstoro) [reply code/api/enrich.py]: Yes, makes sense. https://github.com/CiscoSecurity/tr-05-serverless-recorded-future/pull/4/commits/2c2ace3a0651b7336ab4135295207c265987aaf8

**Review by AnastasiiaRozlyvan — APPROVED** [review]

File: code/container_settings.json [hunk 3]
```diff
@@ -1 +1 @@
-{"VERSION": "1.0.0", "NAME": "Module name"}
+{"VERSION": "1.0.0", "NAME": "Recoded Future"}
```
Comment (TarasMaliuchenko) [inline code/container_settings.json:1]: Typo here. Recor**d**ed Future

Comment (mstoro) [reply code/container_settings.json]: fixed

**Review by TarasMaliuchenko — APPROVED** [review]

**Pull request merged**